WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.m

```diff
@@ -277,6 +282,13 @@ + (NSDictionary *)dictionaryForElement:(id<FBXCElementSnapshot>)snapshot
     };
   }
   
+  // Adding isHittable, only if not excluded
+    if (excludedAttributes == nil || ![excludedAttributes containsObject:FBExclusionAttributeHittable]) {
```

Comment:
this attribute must be excluded by default. Having it in the tree by default would seriously slow down the tree retrieval for all users